src/Project/ToolRunner.ts

```diff
@@ -83,12 +83,10 @@ export class ToolRunner {
     return oneccRealPath;
   }
 
-  public getRunner(name: string, onecc: string, tool: string, toolargs: ToolArgs, path: string) {
+  public getRunner(name: string, tool: string, toolargs: ToolArgs, path: string) {
```

Comment:
This change is necessary because now this `ToolRunner` is very dependent on `onecc` as a driver.